Ginger/GingerCoreNET/Run/GingerExecutionEngine.cs

```diff
@@ -444,8 +446,24 @@ public async Task<int> RunRunnerAsync()
         public void RunRunner(bool doContinueRun = false)
         {
             bool runnerExecutionSkipped = false;
+            List<BusinessFlow>  activeBusinessFlows = new List<BusinessFlow>();
             try
             {
+                if (WorkSpace.Instance.RunsetExecutor.RunSetConfig.ReRunConfigurations.Active && WorkSpace.Instance.RunsetExecutor.RunSetConfig.ReRunConfigurations.RerunLevel == eReRunLevel.BusinessFlow && mGingerRunner != null && WorkSpace.Instance.RunsetExecutor.RunSetConfig.FailedBFGuidList != null && WorkSpace.Instance.RunsetExecutor.RunSetConfig.FailedBFGuidList.Any())
+                {
+                    foreach (BusinessFlow Bf in mGingerRunner.Executor.BusinessFlows)
+                    {
+                        if(WorkSpace.Instance.RunsetExecutor.RunSetConfig.FailedBFGuidList.Contains(Bf.InstanceGuid))
+                        {
+                            activeBusinessFlows.Add(Bf);
+                        }
+                        else
+                        {
+                            Reporter.ToLog(eLogLevel.INFO, $"{Bf.Name} is not failed in refernce ExecutionId {WorkSpace.Instance.RunsetExecutor.RunSetConfig.ReRunConfigurations.ReferenceExecutionID}");
```

Comment:
update message to {Bf.Name} is not failed in refernce ExecutionId {WorkSpace.Instance.RunsetExecutor.RunSetConfig.ReRunConfigurations.ReferenceExecutionID} hence marked inactive for current execution")